Adding OSX builds of IRkernel_0.4.tgz

noamross opened the pull request.

noamross committed:
Adding OSX builds of IRkernel_0.4.tgz

noamross commented:
(A note that things are pretty busy for me right now and you probably don't want to rely on me for this in the near future)

takluyver commented:
Thanks! We can always wait until a Mac user complains and then rope them in to do it ;-)

I think these files are going to be rearranged a bit, so I'll let @JanSchulz take care of it.

jankatins committed:
Import repr tag 0.2

commit 762262c26f143bf3ffc4c74ffe65a50bb3ece9f5
Author: Thomas Kluyver <takowl@gmail.com>
Date:   Thu Jun 11 10:29:07 2015 -0700

    Version number -> 0.2

jankatins committed:
Move macosx packages and add IRkernel_v0.4

jankatins committed:
Add a guide how to add releases

noamross committed:
Merge commit 'refs/pull/13/head' of github.com:IRkernel/irkernel.github.io

noamross committed:
New macosx binaries of repr 0.2
